### PR TITLE
DOCSP-44967: Data modeling in update examples

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-rust/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-44967
Staging - 
https://deploy-preview-143--docs-rust.netlify.app/usage-examples/updateOne/
https://deploy-preview-143--docs-rust.netlify.app/usage-examples/updateMany/
https://deploy-preview-143--docs-rust.netlify.app/usage-examples/replace/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/includes/usage-examples/code-snippets/update-many-async.rs [hunk 1]
```diff
@@ -2,12 +2,27 @@ use std::env;
 use mongodb::{ bson::doc, Client, Collection };
 use bson::Document;
```
Comment (stephmarie17) [inline source/includes/usage-examples/code-snippets/update-many-async.rs:4]: [q] To use Serialize and Deserialize, do the use statements need to be updated?

**Review by stephmarie17 — APPROVED** [review]
LGTM with a question ✅

File: source/usage-examples/replace.txt [hunk 2]
```diff
@@ -26,13 +26,22 @@ Example
...
-documents in the collection.
+the ``sample_restaurants`` database. The ``replace_one()`` method replaces
+the first document in which the value of the ``name`` field is
+``"Landmark Coffee Shop"`` with a new document.
 
-The following code replaces a document in which the value of the
-``name`` field is ``"Landmark Coffee Shop"`` with a new document. MongoDB
-replaces the first document that matches the query filter.
+You can access the documents in the ``restaurants`` collection as instances
+of the BSON data type or a custom data type. To specify which data type represents
```
Comment (isabelatkinson) [inline source/usage-examples/replace.txt:34]:
```suggestion
of the ``Document`` type or a custom data type. To specify which data type represents
```

Ditto elsewhere. This more closely aligns with the code example, but feel free to leave as-is if the existing language is preferred.

**Review by isabelatkinson — APPROVED** [review]
LGTM with one optional suggestion

**Pull request merged**